terraform/scripts/migrate_cluster_psp_to_pss.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+function msg() {
+    if [[ ${VERBOSE} == true ]]; then
+        echo ${@} 2>&1
+    fi
+}
+
+function disable_psp_ns () {
+    local _ns=${1}
+    msg "Disabling PodSecurityPolicy on namespace ${_ns}"
+    kubectl delete -n "${_ns}" rolebinding disable-psp 2>/dev/null
+    kubectl create -n "${_ns}" rolebinding disable-psp \
+            --clusterrole privileged-psp --group "system:serviceaccounts:${_ns}"
+}
+
+function set_pss_label() {
+    local _ns=${1}
+    local _policy=${2}
+    msg "Namespace ${_ns}: setting policy ${_policy}"
+    kubectl label --overwrite ns "${_ns}" "${_policy}"
+}
+
+function set_pss_labels_ns() {
+    local _ns=${1}
+    set_pss_label "${_ns}" "pod-security.kubernetes.io/enforce=privileged"
```

Comment:
for Terraform deployments, can we also do this migration in place? via https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/labels

Reply:
Yes, I'm looking into doing that with Terraform.